verify/angular/src/app/app.component.ts

```diff
@@ -0,0 +1,34 @@
+/* --------------------------------------------------------------------------------------------
+ * Copyright (c) 2024 TypeFox GmbH (http://www.typefox.io). All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ * ------------------------------------------------------------------------------------------ */
+
+import { AfterViewInit, Component } from '@angular/core';
+import { startEditor } from 'monaco-languageclient-examples';
+import { useWorkerFactory } from 'monaco-editor-wrapper/workerFactory';
+import { jsonClientUserConfig } from 'monaco-languageclient-examples/json-client';
+
+useWorkerFactory({
+    rootPath: window.location.href + '../..',
+    basePath: '../assets',
+});
+
+const codeMain = `{
+    "$schema": "http://json.schemastore.org/coffeelint",
+    "line_endings": {"value": "windows"}
+}`;
+
+@Component({
+    selector: 'app-root',
+    templateUrl: './app.component.html',
+    styleUrls: ['./app.component.css']
+})
+export class MonacoEditorComponent implements AfterViewInit {
+    title = 'angular-client';
+    initDone = false;
+
+    async ngAfterViewInit(): Promise<void> {
+        const htmlElement = document.getElementById('monaco-editor-root');
+        startEditor(jsonClientUserConfig, htmlElement, codeMain);
```

Comment:
is the https://github.com/TypeFox/monaco-languageclient-ng-example repo merged into this monorepo ?
can we add a `console.log` in this component to demo how to read the edited value back from the editor ?

thx,

Reply:
> is the https://github.com/TypeFox/monaco-languageclient-ng-example repo merged into this monorepo ?

Yes, once this PR is merged. The need for the external repo is removed.

> how to read the edited value back from the editor ?

This is done by the react component, see here: https://github.com/TypeFox/monaco-languageclient/blob/merge-monaco-edtior-wrapper/packages/wrapper-react/src/index.tsx#L171-L191 we could also think about moving or having this function directly in the wrapper, but if you need it quickly, you can just use this as a starting point.